bindings/gumjs/gumdukcore.c

```diff
@@ -2816,50 +2824,85 @@ gum_duk_native_function_invoke (GumDukNativeFunction * self,
                                 duk_idx_t argv_index)
 {
   GumDukCore * core;
-  gsize nargs;
+  ffi_cif *cif;
   ffi_type * rtype;
+  ffi_type ** atypes;
+  gsize nargs, nargs_fixed;
+  gboolean is_variadic;
```

Comment:
Variables should be in the same order that they're first used in. (Except if one holds the return value, in which case it should be first.)

Reply:
Ok, i will fix it.